### ISSUE TITLE
HMW commercialize 0xchat

### ISSUE THREAD

**Issue by alltheseas** [0xchat-app/0xchat-core#9]
How might we commercialize 0xchat?

1) add subscription option (see signal)
2) add one time top-up model (see mullvad VPN)

What are the benefits of paid users:
1) a star (⭐️) or other subscriber only indicator
2) custom 0xchat supporter skin cc @karnagebitcoin 
3) premium only feature - e.g. search all my notes

**Comment by water783** [comment]
Very good advice, I have also been thinking about some ways about commercializing 

We have designed badges, and perhaps I will also consider some ways to play with badges.

**Comment by karnagebitcoin** [comment]
Sorry, I've been out of the loop for too long. Some questions:

1. Whats the primary use case of this app?
2. Who is the primary user (at least initially?)
3. What does that user care about most?

**Comment by water783** [comment]
The goal of 0xchat is to build a chat app that protects privacy and resists censorship, combining interesting features with the BTC payments. 

Therefore, I think the main users are privacy-conscious Bitcoiners who can happily find companions, chat, maintain privacy, and use Lightning/Ecash payments here.

**Comment by karnagebitcoin** [comment]
Ok thanks. It's a bit hard for me to recommend any way to monetize it without using it every day. 

Generally speaking you want to paygate some nice feature that everyone wants, but typically would be considered outside the scope of normal function. (ie not essential to using the app, but so nice that people really want it or need it). 

Depending on how aggressive you want to be, you may want to push the premium upsell at onboarding. I would check out how other (Popular) apps do upsells in onboarding and mimic that at first. 

I had a friend who made iOS apps for a living, and he said monetizing them is really difficult. People generally do not want to pay anything to use an app, but it's not impossible. Plenty of good apps charge and people pay. 

TLDR; 
- show a great feature and let people use it, but unable to finish using it * require an upgrade to PRO
- Check out other successful apps to see what they do.

**Comment by water783** [comment]
yeah, this is just a discussion, but I think there's a long way to go for commercialization. Firstly, we need more users to use our product every day, and then comes the premium services.

**Comment by alltheseas** [comment]
Thanks for the comment @karnagebitcoin 

@water783 one thing I learned from Damus is that there is a subset of users who want the devs to make easy to donate. They are happy to pay for new features, and appreciate something trivial like a star or rocket that signifies their supporter status in front of other chat app users.

Don't be afraid to roll this out, even if you deem the user count low.

**Comment by water783** [comment]
Thx let me know, i will give it a try. :)

**Comment by water783** [comment]
Is there a NIP about subscription btw? 🤔

**Comment by alltheseas** [comment]
> Is there a NIP about subscription btw? 🤔

Good question - I dont know. 

We'll have it soon on Damus - part iOS (fiat), part web (sats) if you want to see how it was approached.

**Comment by MrZarp** [comment]
premium feature idea: support for notes, flare.pub videos, zap.stream streams, longform content, stemstr tracks, lists, calendar events, geolocations, highlights directly in the chat window.  In order for the event to be displayed directly in the chat window, the sender must have a premium badge.

**Comment by MrZarp** [comment]
another idea: premium user can add a link list to its profile

**Comment by MrZarp** [comment]
here's another: a premium user can add the notes feed of the person he is following to the chat list - it will look similar to WhattsApp channels.

**Comment by water783** [comment]
@MrZarp  thank you for your suggestion, it's very enlightening 😄
